### ISSUE TITLE
Copyright infringement issue

### ISSUE THREAD

**Issue by Ahwxorg** [libreddit/libreddit#848]
Hey there,

Not really sure how to open this issue, but: I've just gotten this email from COPYTRACK:

![image](https://github.com/libreddit/libreddit/assets/87855602/11180dfa-afc0-4a24-a1a7-af85feba8dd2)
![image](https://github.com/libreddit/libreddit/assets/87855602/2538108f-35bf-49fc-b717-f68a7c34baf4)

Questions:

* Am I fine? If not, what do I do?
* How do I reply to this?
* Is there a way to prevent this from happening in the future?

For now my instance is down.

Please help me...!

**Comment by throwaway-d** [comment]
@libreddit

**Comment by Midou36O** [comment]
Feel free to use the template [available at invidious docs](https://docs.invidious.io/takedown/) and to reword it to be used for libreddit.

**Comment by throwaway-d** [comment]
Should we also make a template to add into Libreddit Wiki?
